src/parse_utils.rs

```diff
@@ -170,3 +176,64 @@ pub(crate) fn consume_comma(tokens: &mut TokenIter) -> Option<Punct> {
         _ => None,
     }
 }
+
+/// Parse `::`, as in path separator or turbofish.
+///
+/// Does not advance `tokens` if the double colon is not found.
+pub(crate) fn try_consume_colon2(tokens: &mut TokenIter) -> Option<[Punct; 2]> {
```

Comment:
Should be `consume_colon2` (or consume_double_colon?)

Reply:
Done.